pym/bob/input.py

```diff
@@ -911,8 +911,91 @@ def hasJenkinsPlugin(self):
         return False
 
     def status(self, workspacePath, dir, verbose = 0):
-        print("CVS SCM status not implemented!")
-        return 'error'
+        workDir = os.path.join(workspacePath, dir)
+        cvsDir = os.path.join(workDir, 'CVS')
+        if not os.path.exists(workDir) or not os.path.exists(cvsDir):
+            return 'error'
```

Comment:
Likewise, I copied this logic from SvnScm which returns "error" if it cannot find the directory.

    def status(self, workspacePath, dir, verbose=0):
        scmdir = os.path.join(workspacePath, dir)
        if not os.path.exists(os.path.join(os.getcwd(), scmdir)):
            return 'error'

Looking forward to an interface description of the `status` method :-)))